<ui version="4.0" >
 <class>FilterDetailsWidget</class>
 <widget class="QWidget" name="FilterDetailsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Filter</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>2</number>
   </property>
   <property name="leftMargin" >
    <number>2</number>
   </property>
   <property name="topMargin" >
    <number>2</number>
   </property>
   <property name="rightMargin" >
    <number>2</number>
   </property>
   <property name="bottomMargin" >
    <number>2</number>
   </property>
   <item>
    <widget class="QToolBox" name="toolBox" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="settingsPage" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>755</width>
        <height>705</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Settings</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="nameLabel" >
         <property name="text" >
          <string>Name</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="name" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QLabel" name="typeLabel" >
         <property name="text" >
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" >
        <widget class="QComboBox" name="type" />
       </item>
       <item row="0" column="4" >
        <widget class="QLabel" name="testStringLabel" >
         <property name="text" >
          <string>Test String</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="5" >
        <widget class="QLineEdit" name="testString" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>4</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="6" >
        <widget class="KEditListBox" name="expressions" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Expressions</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading</set>
         </property>
         <property name="flat" >
          <bool>false</bool>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
         <property name="buttons" >
          <set>KEditListBox::Add|KEditListBox::Remove</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="6" >
        <widget class="QGroupBox" name="multiMatchBox" >
         <property name="title" >
          <string>MultiMatch</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="multiMatchLabel" >
            <property name="text" >
             <string>MultiMatch</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="multiMatch" />
          </item>
          <item row="0" column="2" >
           <widget class="QLabel" name="rereleaseLabel" >
            <property name="text" >
             <string>Rerelease</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QComboBox" name="rerelease" />
          </item>
          <item row="0" column="4" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" colspan="6" >
           <widget class="KEditListBox" name="captureExpressions" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title" >
             <string>Capture Expressions</string>
            </property>
            <property name="flat" >
             <bool>false</bool>
            </property>
            <property name="buttons" >
             <set>KEditListBox::Add|KEditListBox::Remove</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="6" >
           <widget class="QGroupBox" name="captureChecksBox" >
            <property name="title" >
             <string>Capture Checks</string>
            </property>
            <layout class="QGridLayout" >
             <property name="leftMargin" >
              <number>2</number>
             </property>
             <property name="topMargin" >
              <number>2</number>
             </property>
             <property name="rightMargin" >
              <number>2</number>
             </property>
             <property name="bottomMargin" >
              <number>0</number>
             </property>
             <property name="horizontalSpacing" >
              <number>2</number>
             </property>
             <property name="verticalSpacing" >
              <number>2</number>
             </property>
             <item row="0" column="0" >
              <widget class="QTableView" name="captureCheckTableView" />
             </item>
             <item row="0" column="1" >
              <widget class="KToolBar" native="1" name="captureCheckToolbar" />
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="5" >
           <widget class="KToolBar" native="1" name="multiMatchToolbar" />
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="6" >
        <widget class="QGroupBox" name="sourceListBox" >
         <property name="title" >
          <string>Source List</string>
         </property>
         <layout class="QGridLayout" >
          <property name="leftMargin" >
           <number>2</number>
          </property>
          <property name="topMargin" >
           <number>2</number>
          </property>
          <property name="rightMargin" >
           <number>2</number>
          </property>
          <property name="bottomMargin" >
           <number>0</number>
          </property>
          <property name="horizontalSpacing" >
           <number>2</number>
          </property>
          <property name="verticalSpacing" >
           <number>2</number>
          </property>
          <item row="0" column="0" colspan="2" >
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="sourceListTypeLabel" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>Source List Type</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="sourceListType" />
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>371</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="0" >
           <widget class="QListView" name="sourceListView" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="KToolBar" native="1" name="sourceListToolbar" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="matchesPage" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>94</width>
        <height>72</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Matches</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QTableView" name="matchesTableView" />
       </item>
       <item>
        <widget class="QWidget" native="1" name="matchesToolbar" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
  </customwidget>
  <customwidget>
   <class>KToolBar</class>
   <extends>QWidget</extends>
   <header>ktoolbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
